tests/thread_priority_inversion_msg/README.md

```diff
@@ -0,0 +1,44 @@
+NOTE
+====
+THIS TEST WILL FAIL WHEN RUNNING RIOT USING THE DEFAULT CORE/KERNEL CONFIGURATION!
+
+TODO: merge the `core_priority_inheritance` module to enable priority
+      inheritance and with this successfully pass this test.
+
+Expected result
+===============
+When successful, you will see the 3 configured threads printing 6 different
+events in a defined order. If the test passes, the output should look like this:
+
+```
+main(): This is RIOT! (Version: xxx)
+Test for showing priority inversion when using msg_send_receive
+
+If this tests succeeds, you should see 6 events appearing in order.
+The expected output should look like this:
+Event  1:      t3 - waiting for incoming message
+Event  2:      t2 - starting infinite loop, potentially starving others
+Event  3:      t1 - sending msg to t3 (msg_send_receive)
+Event  4:      t3 - received message
+Event  5:      t3 - sending reply
+Event  6:      t1 - received reply
```

Comment:
I reckon this is starvation, rather than priority inversion. Reason is that the medium priority thread prevents the low thread from running even before the message is sent, rather than pre-empting it while it's handling the message. Priority inversion would involve the second one - i.e. indirect pre-emption of a higher priority thread.
An example displaying priority inversion with messages would involve the high thread sending the message to the low thread, and the low thread receiving it and doing some action before sending. But while that action is happening, the medium thread comes along and pre-empts the low thread.

Reply:
yes, will fix.